index.html

```diff
@@ -65,190 +70,239 @@ <h2 class="icon-text">LET'S CONNECT</h2>
+           </ul>
+          </li>
+          </ul>
+          <br />
+          <ul class="frame-works">
+            <li class="languages-down languages-down-line">
+              <h3 class="space-down">Frameworks</h3>
+              <i class="fa fa-angle-right" aria-hidden="true"></i>
+            </li>
+            <li class="languages-down languages-down-line">
+              <h3 class="space-down">Skills</h3>
+              <i class="fa fa-angle-right" aria-hidden="true"></i>
             </li>
           </ul>
-          <button type="button" class="about-btn">Get My Resume</button>
         </div>
-        <ul class="tech-space">
-          <li class="languages-down tech-space">
-            <h3>Languages</h3>
-            <i class="fa fa-angle-down" aria-hidden="true"></i>
-          </li>
-          <li class="dropdown-technologies">
-            <img src="./Images/javascript.png" alt="javaScript" />
-            <p>Javascript</p>
-          </li>
-          <li class="dropdown-technologies">
-            <img src="./Images/HTML.png" alt="html" />
-            <p>HTML</p>
-          </li>
-          <li class="dropdown-technologies">
-            <img src="./Images/CSS.png" alt="css" />
-            <p>CSS</p>
-          </li>
-        </ul>
-        <br />
-        <ul class="frame-works">
-          <li class="languages-down languages-down-line">
-            <h3 class="space-down">Frameworks</h3>
-            <i class="fa fa-angle-right" aria-hidden="true"></i>
-          </li>
-          <li class="languages-down languages-down-line">
-            <h3 class="space-down">Skills</h3>
-            <i class="fa fa-angle-right" aria-hidden="true"></i>
-          </li>
-        </ul>
       </section>
     </main>
-     <!-- footer -->
-  <footer class="footer-container">
-    <div class="contact-wrapper">
-      <div class="contact-wrapper-two">
-        <div class="contactme-body">
-          <h2 class="contact-heading">Contact me</h2>
-          <p class="message-contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <!-- footer -->
+    <footer class="footer-container">
+      <div class="contact-wrapper">
+        <div class="contact-wrapper-two">
+          <div class="contactme-body">
+            <h2 class="contact-heading">Contact me</h2>
+            <p class="message-contact">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <form
+            action="https://formspree.io/f/xeqnjrlj"
+            method="post"
+            class="contact-form"
+          >
+            <input
+              type="text"
+              name="user_name"
+              placeholder="Name(s)"
+              maxlength="30"
```

Comment:
- [ ] KIndly follow the design template for the form section also.
- [ ] It would be nice to also use the `background-images` used for this section. That way, you will make your design look like that of Figma's 🙏 

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144683950-fbeecd4a-9190-4b70-ba41-32be2a98998d.png)

### Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144683982-cfd6ac93-5ce6-4073-9dab-65c0f35b8f42.png)